conftest.py

```diff
@@ -1,10 +1,26 @@
+import os
 import pytest
 
 from openpilot.common.prefix import OpenpilotPrefix
 
 
 @pytest.fixture(scope="function", autouse=True)
-def global_setup_and_teardown():
+def openpilot_function_fixture():
+  starting_env = dict(os.environ)
+
   # setup a clean environment for each test
   with OpenpilotPrefix():
     yield
+
+  os.environ.clear()
+  os.environ.update(starting_env)
+
+
+@pytest.fixture(scope="class", autouse=True)
```

Comment:
why do we need two fixtures for this?

Reply:
if you use setUpClass, the environment variable will persist

Reply:
so you need to clear the environment variables before each test function, and each class

Reply:
ah, can you leave a comment explaining this? it's not obvious from reading this

Reply:
added a comment